index.htm

```diff
@@ -150,7 +153,7 @@
 
             var yAxis = new Rickshaw.Graph.Axis.Y({
                 graph: graph,
-                tickFormat: Rickshaw.Fixtures.Number.formatKMBT,
+                tickFormat: function(x) { return (x * 100) + '%'; },
```

Comment:
nit: shouldn't we use the yFormatter ? Or at least the same formatting (i.e., with toFixed(2) ?)